<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文献管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/nicon/search.png</normaloff>:/nicon/search.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/nicon/DailyBing-zh-cn-20250519-FHD.jpg);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0">
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="3" column="0">
     <widget class="QGroupBox" name="LabelBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>索引：</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="b_Key">
         <property name="text">
          <string>关键词</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QRadioButton" name="b_Author">
         <property name="text">
          <string>作者</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="b_Title">
         <property name="text">
          <string>题目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="b_Coauthor">
         <property name="text">
          <string>合著者</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="b_Fuzzy">
         <property name="text">
          <string>模糊搜索</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="authorPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTableWidget" name="Table_1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>391</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>811</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>标题</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>作者</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>最后修改日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>出版社</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>关键词</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="titlePage">
       <widget class="QLabel" name="Label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>801</width>
          <height>391</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="coauthorPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTextBrowser" name="Browser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>801</width>
          <height>391</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="fuzzyPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QTableWidget" name="Table_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>801</width>
          <height>401</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Modify Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Key</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="TextLine">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fuzzy">
        <property name="text">
         <string>模糊搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Search">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/search.png</normaloff>:/search.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="a_Open"/>
    <addaction name="a_Status"/>
    <addaction name="a_Exit"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="a_Clear"/>
    <addaction name="a_AuthorCount"/>
    <addaction name="a_FuzzyOpen"/>
    <addaction name="a_GroupAnalyze"/>
    <addaction name="a_KeyWordCount"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="a_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFunction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100,100, 200);</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="a_Open"/>
   <addaction name="a_Status"/>
   <addaction name="a_Clear"/>
   <addaction name="a_FuzzyOpen"/>
   <addaction name="a_AuthorCount"/>
   <addaction name="a_GroupAnalyze"/>
   <addaction name="a_KeyWordCount"/>
   <addaction name="a_Help"/>
   <addaction name="a_Exit"/>
  </widget>
  <action name="a_Help">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/help_filled.png</normaloff>:/nicon/help_filled.png</iconset>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="a_Open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/store.png</normaloff>:/nicon/store.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="a_Exit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/close.png</normaloff>:/nicon/close.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="a_Clear">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/ashbin.png</normaloff>:/nicon/ashbin.png</iconset>
   </property>
   <property name="text">
    <string>清除文件</string>
   </property>
  </action>
  <action name="a_AuthorCount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/user.png</normaloff>:/nicon/user.png</iconset>
   </property>
   <property name="text">
    <string>作者统计</string>
   </property>
  </action>
  <action name="a_FuzzyOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/flow.png</normaloff>:/nicon/flow.png</iconset>
   </property>
   <property name="text">
    <string>模糊匹配</string>
   </property>
  </action>
  <action name="a_GroupAnalyze">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/sub_account.png</normaloff>:/nicon/sub_account.png</iconset>
   </property>
   <property name="text">
    <string>聚团分析</string>
   </property>
  </action>
  <action name="a_KeyWordCount">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/count.png</normaloff>:/nicon/count.png</iconset>
   </property>
   <property name="text">
    <string>热词统计</string>
   </property>
  </action>
  <action name="a_Status">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/nicon/checksearch.png</normaloff>:/nicon/checksearch.png</iconset>
   </property>
   <property name="text">
    <string>查询状态</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
